src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -408,46 +352,8 @@ namespace OpenLoco
             addr<0x005233B2, int32_t>() += addr<0x01140840, int32_t>();
             addr<0x0114084C, int32_t>() = 0;
             addr<0x01140840, int32_t>() = 0;
-            if (Config::get().old.var_72 == 0)
-            {
-                Config::get().old.var_72 = 16;
-                const auto cursor = Ui::getCursorPosScaled();
-                addr<0x00F2538C, Ui::Point32>() = cursor;
-                drawingCtx.clear(0);
-                addr<0x00F2539C, int32_t>() = 0;
-            }
-            else
-            {
-                if (Config::get().old.var_72 >= 16)
```

Comment:
What even is this code doing?

Reply:
It puzzled me as well. I _think_ it may be a makeshift delay to wait for the initialisation of the DirectX surface to finish before locking it...?